dht-node/src/dht_node/index.ts

```diff
@@ -29,14 +24,20 @@ export type CommunityApi = {
   url: string
 }
 
+type KeyPair = { publicKey: Buffer; secretKey: Buffer }
+
 export const startDHT = async (topic: string): Promise<void> => {
   try {
     const TOPIC = DHT.hash(Buffer.from(topic))
-    const keyPair = DHT.keyPair(getSeed())
+    const keyPair = DHT.keyPair(
```

Comment:
it seems not uptodate with the master, the comment about the logic is missing here

Reply:
![image](https://github.com/gradido/gradido/assets/1238238/3886c9d4-69d0-4e9b-a3da-48a6cbed386e)